frontend/src/components/PromptSpeech.tsx

```diff
@@ -15,16 +14,17 @@ const SpeechContainer = styled.div`
 
 interface SpeechProp {
   prompt: string;
+  button: ReactElement;
```

Comment:
I wonder if we should make it like use the `children` input, so that like the PromptSpeech is like a wrapper around its component. What do you think?

Reply:
Hmm i think for promptSpeech it sounds more like component to me rather than a layout/wrapper cuz i dont think we need to do something like <PromptSpeech>blah blah </PrompSpeech> since technically only the prompt text is changing

Reply:
Ok sounds good!